src/sha1.cpp

```diff
@@ -75,12 +75,11 @@ namespace
 	template <class BlkFun>
 	void SHA1transform(u32 state[5], u8 const buffer[64])
 	{
-		using namespace std;
-		u32 a, b, c, d, e;
+		std::u32 a, b, c, d, e;
```

Comment:
Actually it's a typedef at top of file `typedef boost::uint32_t u32;` so I should revert that change.